parquet/src/file/metadata.rs

```diff
@@ -353,6 +377,15 @@ impl RowGroupMetaDataBuilder {
         self
     }
 
+    /// Sets the sorting order for columns
+    pub fn set_sorting_columns(
+        mut self,
+        value: Option<Vec<SortingColumnMetaData>>,
+    ) -> Self {
+        self.sorting_columns = value;
```

Comment:
```suggestion
        value: Vec<SortingColumnMetaData>,
    ) -> Self {
        self.sorting_columns = Some(value);
```
This is consistent with set_page_offset

Reply:
I got a reverse review before. The reason given by the reviewer was that if we remove `Option` from signature then the function cannot be used to set `None` for this field. I am going to keep this as is.